File: app/src/main/java/basic/ex03.java
package basic;

public class ex03 {
    public static void main(String[] args) {
        int height = 5;

        for (int i = 0; i < height; i++) {
            for (int s = 0; s < height - i - 1; s++) {
                System.out.print(" ");
            }
            for (int star = 0; star < i * 2 + 1; star++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: app-kotlin/src/main/kotlin/basic/ex03.kt
package basic

class ex03 {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val height = 5
            for (i in 0 until height) {
                print(" ".repeat(height - i - 1))
                println("*".repeat(i * 2 + 1))
            }
        }
    }
}
